lib/metanorma/ogc/processor.rb

```diff
@@ -25,8 +25,8 @@ def fonts_manifest
           "Arial" => nil,
           "STIX Two Math" => nil,
           "Source Han Sans" => nil,
+          "Source Han Sans Normal" => nil,
           "Fira Code" => nil,
```

Comment:
@Intelligent2013 Do we need "Fira Code" subfamilies? "Fira Code Light"

Reply:
No, we use 'style="normal" weight="normal"' only.

Reply:
Ok, thank you :)